test/arguments.ts

```diff
@@ -29,6 +29,11 @@ test('`url` should be utf-8 encoded', async t => {
 	);
 });
 
+test('throws if no arguments provided', async t => {
+	// @ts-ignore This is on purpose.
+	await t.throwsAsync(got(), TypeError, 'Missing `url` argument');
```

Comment:
```suggestion
	await t.throwsAsync(got(), {
		instanceOf: TypeError,
		message: 'Missing `url` argument'
	});
```

Reply:
Meh, I did this a few times 😅

Reply:
Do I really need to change this? It looks so slim...

Reply:
You don't have to. Was just a suggestion to improve readability.